src/state.rs

```diff
@@ -570,6 +575,11 @@ impl DemandProxyState {
 
         // Allow and deny logic follows https://istio.io/latest/docs/reference/config/security/authorization-policy/
 
+        for pol in deny_dry_run.iter() {
+            if pol.matches(conn) {
+                debug!(policy = pol.to_key().as_str(), "dry-run deny policy match");
```

Comment:
I've always found it curious that these are debug logs in envoy.

Reply:
makes them kind of ugly too
```
2025-12-31T14:09:03.793034Z     debug   state:proxy{wl=server/httpbin-654df84766-hwj4m}:inbound{id=1f022ae2187b6d519923ca493b1dbaec peer=10.244.1.4:46190}      dry-run deny policy match       policy="server/dry-run-example"
```